opencti-platform/opencti-graphql/src/parser/csv-mapper.ts

```diff
@@ -227,21 +230,68 @@ const handleAttributes = (
       // Handle column attribute
       handleDirectAttribute(attribute, input, record, attributeDef);
     } else if (refDef || ['from', 'to'].includes(attribute.key)) {
-      // Handle based_on attribute
       handleBasedOnAttribute(attribute, input, refDef, otherEntities, refEntities);
     } else {
       throw UnsupportedError('Unknown schema for attribute:', { attribute });
     }
   });
 };
 
+/**
+ * We handle markings in a specific function instead of doing it inside the
+ * handleAttributes() one because we need to do specific logic for this attribute.
+ */
+const handleDefaultMarkings = (
+  entitySetting: BasicStoreEntityEntitySetting | undefined,
+  representation: CsvMapperRepresentation,
+  input: Record<string, InputType>,
+  refEntities: Record<string, BasicStoreObject>,
+  chosenMarkings: string[],
+  user: AuthUser,
+) => {
+  if (input[INPUT_MARKINGS]) {
+    return;
+  }
+
+  // Find default markings policy in entity settings ("true" or undefined).
+  const settingAttributes = entitySetting ? getAttributesConfiguration(entitySetting) : undefined;
+  // console.log('pouet', representation.target.entity_type, settingAttributes);
+  const settingMarkingValue = settingAttributes
+    ?.find((attribute) => attribute.name === INPUT_MARKINGS)
+    ?.default_values?.[0];
+  // Find default markings policy in mapper representation ("user-default" or "user-choice"  or undefined).
+  const representationMarkingValue = representation.attributes
+    .find((attribute) => attribute.key === INPUT_MARKINGS)
+    ?.default_values?.[0];
+
+  // Retrieve default markings of the user.
+  const userDefaultMarkings = (user.default_marking ?? [])
+    .find((entry) => entry.entity_type === 'GLOBAL')
```

Comment:
I wonder if we don't already have a constant somewhere for this 'GLOBAL' string.

Reply:
I can't find one when looking into the backend code, but could be nice to create one